Lib/glyphsLib/parser.py

```diff
@@ -137,21 +141,25 @@ def _unescape_fn(m):
             return unichr(int(m.group(1)[1:], 8))
         return unichr(int(m.group(2)[2:], 16))
 
-    def _trim_value(self, value):
+    @staticmethod
+    def unescape_text(text):
         """Trim double quotes off the ends of a value, un-escaping inner
-        double quotes.
-        Also convert escapes to unicode.
-        """
+        double quotes.  Also convert escapes to unicode."""
+
+        if text[0] == '"':
+            assert text[-1] == '"'
+            text = text[1:-1].replace('\\"', '"')
+        return Parser._unescape_re.sub(Parser._unescape_fn, text)
+
+
+    def _trim_value(self, value):
+        return Parser.unescape_text(value) if self.unescape else value
```

Comment:
here you may use `self.unescape_text` instead of `Parser.unescape_text` (e.g. a subclass could potentially override the static method).